styles2.css

```diff
@@ -657,3 +542,267 @@ footer > div.contentline {
+
+  section.recentWork {
+    width: auto;
+    height: auto;
+    padding: 84px 8px;
+    display: grid;
+    grid-template-columns: 369px 369px 369px;
+    align-items: center;
+    margin-left: auto;
+    margin-right: auto;
+    justify-content: space-between;
+    row-gap: 24px;
+    column-gap: 32px;
+    margin-bottom: 28px;
+  }
+
+  div.title {
+    grid-column: 1/span 3;
+    margin-bottom: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: -4px;
+  }
+
+  section.aboutMe > article > div,
+  section.aboutMe > article > p {
+    max-width: 920px;
+    width: 64%;
+  }
+
+  section.recentWork > div.rHeader.title > div {
+    display: none;
+  }
+
+  h1.title > br {
+    display: none;
+  }
+
+  section.recentWork > article {
+    max-width: 369px;
+  }
+
+  ul.language {
+    width: auto;
+  }
+
+  .vide {
+    height: 252px;
+  }
+
+  section.aboutMe {
+    height: auto;
+    padding: 80px 0;
+    column-gap: 48px;
+  }
+
+  section.aboutMe > article.text {
+    width: inherit;
+    background-image: url("illustration\ about\ me\ desktop\ \(2\).svg"), url(illustration-desktopText.svg);
+    background-position: top right, bottom 16px left;
+    background-size: contain, auto;
+    flex-direction: column;
+    align-items: center;
+    align-content: space-around;
+    gap: 24px;
+  }
+
+  section.aboutMe > article > p {
+    height: auto;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  section.aboutMe > article.text > button {
+    margin-top: 0;
+  }
+
+  ul.maintain {
+    margin-top: 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 8px;
+    gap: 24px;
+    width: 1171px;
+    height: auto;
+  }
+
+  div.know {
+    width: 100%;
+    height: 352px;
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    gap: 0;
+    padding: 0 16px;
+    padding-bottom: 24px;
+  }
+
+  div.know > div.contain {
+    margin-top: 0;
+    margin-bottom: 64px;
+    padding-bottom: 0;
+  }
+
+  div.know > div.contain > div.empty {
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  div.know > div.describe {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    margin-bottom: 0;
+  }
+
+  /* Hide footer mobile */
+
+  .mobile {
+    display: none;
+  }
+
+  .desktop {
+    display: block;
+  }
+
+  .contact-form {
+    display: grid;
+    grid-template-columns: 436px 1fr;
+    align-items: flex-start;
+    padding: 80px 0;
+    gap: 24px;
+    width: 1171px;
+    margin: 0 auto 145px auto;
+    background-image: none;
+  }
+
+  .deskcol {
+    grid-row: 1/span 3;
+  }
+
+  .contact-form > form {
+    margin-left: 0;
+  }
+
+  .contact-form > form > input {
+    height: 48px;
+  }
+
+  .contact-form > form > textarea {
+    height: 114px;
+  }
+
+  .icon.desktop > ul {
+    margin: 15px auto;
+    width: 100%;
+  }
+
+  hr {
+    height: 1px;
+    color: #dfe1e6;
+  }
```

Comment:
- [ ] I have noticed that the `hr` does not cover `100% width` of the `body` :disappointed: . You should ensure that the line sticks to the width of the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=Kv8PktMsmpk2hK5U-0). :+1: 
- Your design :sunglasses: 
![Screenshot from 2023-01-20 06-34-29](https://user-images.githubusercontent.com/99394374/213612183-9008c944-cf3c-4fde-a00c-28c7b9e317e2.png)